.github/pull_request_template.md

```diff
@@ -17,4 +17,4 @@ Relates to #
 ## Checklist before merging
 
 - [ ] Test, docs, adr added or updated as needed
-- [ ] [Contributor Guide Steps](https://github.com/justinthelaw/repository-template/blob/main/.github/CONTRIBUTING.md) followed
+- [ ] [Contributor Guide Steps](CONTRIBUTING.md) followed
```

Comment:
```suggestion
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/pkg/blob/main/CONTRIBUTING.md) followed
```

I think this needs to be a absolute URL here, pretty sure the link will fail in the GitHub UI otherwise